contrib/devtools/check-deps.sh

```diff
@@ -32,32 +31,28 @@ ALLOWED_DEPENDENCIES=(
     "wallet common"
     "wallet crypto"
     "wallet util"
-    "wallet_tool util"
-    "wallet_tool wallet"
 )
 
 # Add minor dependencies omitted from doc/design/libraries.md to keep the
 # dependency diagram simple.
 ALLOWED_DEPENDENCIES+=(
     "wallet consensus"
-    "wallet_tool common"
-    "wallet_tool crypto"
```

Comment:
What happened to **wallet_tool** in the latest push?

2 dependencies exist according to **/doc/design/libraries.md**: 
```
libbitcoin_wallet_tool-->libbitcoin_wallet;
libbitcoin_wallet_tool-->libbitcoin_util;
```

Reply:
I didn't dig into it but it appeared wallet_tool library disappeared in the conversion to cmake, so I removed references to it here. Probably libraries.md needs to be updated too though. There may be an existing cmake documentation PR where that change could be / has been made

Reply:
> I didn't dig into it but it appeared wallet_tool library disappeared in the conversion to cmake...

I couldn’t bring myself to create a library for just a single object file. :)

For the reference, the library was defined as follows: https://github.com/bitcoin/bitcoin/blob/80f00cafdeef0600fa1a5e906686720786c2336c/src/Makefile.am#L547-L549